### ISSUE TITLE
Problems with socket.io

### ISSUE THREAD

**Issue by tamasgal** [NarrativeScience/log.io#184]
I am not a nodejs expert at all, so I am not sure how this is related to log.io or socket.io, or an incompatibility of my nodejs installation.

I was running log.io-server without any problems and after a a few weeks I wanted to power it on again, however now it shows this error when I want to launch it:

```
tgal@pi2089:~
19:46:21 > log.io-server
/usr/local/lib/node_modules/log.io/node_modules/socket.io/lib/store.js:35
Store.prototype.__proto__ = EventEmitter.prototype;
                                        ^

TypeError: Cannot read property 'prototype' of undefined
    at Object.<anonymous> (/usr/local/lib/node_modules/log.io/node_modules/socket.io/lib/store.js:35:41)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/log.io/node_modules/socket.io/lib/manager.js:16:13)
    at Module._compile (module.js:573:32)
```

I guess nodejs and/or socket.io was updated during the last weeks (I am running Mac OS X 10.9). The nodejs/npm version is:

```
tgal@pi2089:~
19:46:37 > node -v
v8.0.0-head
tgal@pi2089:~
19:47:23 > npm --version
3.10.8

socket.io@1.5.0
```

Is log.io not compatible with this version, or is any compatibility issue with the latest socket.io?

Any help would be appreciated. I really love log.io and I need it ;)

**Comment by tamasgal** [comment]
I installed the latest nodejs v6 and now it works again!

This is obviously the reason why it is not running on v8:

```
   info  - socket.io started
(node:58548) DeprecationWarning: process.EventEmitter is deprecated. Use require('events') instead.
```

**Comment by msmathers** [comment]
This should be resolved using v0.4.3
